intellij/src/saros/intellij/ui/views/buttons/SimpleButton.java

```diff
@@ -1,17 +1,30 @@
 package saros.intellij.ui.views.buttons;
 
+import com.intellij.openapi.project.Project;
 import javax.swing.ImageIcon;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
 import saros.intellij.ui.actions.AbstractSarosAction;
 
 /** Simple button used to create actions that just call {@link AbstractSarosAction#execute()}. */
 public class SimpleButton extends AbstractToolbarButton {
```

Comment:
Good candidate for renaming. Isn't ActionToolbarButton a better name ?

Reply:
Probably. Also, it is a good canidate for removal. It was basically only used to disaply buttons without an implementation, which are currently not shown anyways. So just throwing it away and replacing the usage with a TODO "add this button" would be for the best in my opinion.